upgrades/upgrade_testing_framework/steps/step_restore_source_snapshot.py

```diff
@@ -1,58 +1,48 @@
 import json
 import time
 
+import upgrade_testing_framework.clients as clients
 from upgrade_testing_framework.core.framework_step import FrameworkStep
-import upgrade_testing_framework.core.shell_interactions as shell
 
 class RestoreSourceSnapshot(FrameworkStep):
```

Comment:
What is the plan for this step once we take out the current `test_source` and `test_target`  steps? I agree we should be making sure the snapshot exists and is restored, but do we plan to do any document checking like we are now or do we shift that responsibility to the robot tests